だよ〜ん caused infinite loop

The code in calctrans/* should be updated to cope with 
such situation.

But that is quite invasive. For now, let's work around
by changing test data.

diff --git a/corpus/corpus.3.txt b/corpus/corpus.3.txt
index fe694fb..f2c3156 100644
--- a/corpus/corpus.3.txt
+++ b/corpus/corpus.3.txt
@@ -7098,7 +7098,7 @@ Rawでなげたら |Rawで|なげたら| |Rawで|投げたら|
 |あっぷぐれーど|よう| |アップグレード|用|
 |ねっとわーく|いんたーふぇーす| |ネットワーク|インターフェース|
 |せっちなどの|じゅんび|さぎょうは| |設置などの|準備|作業は|
-|あっぷでーと|だよ〜ん| |アップデート|だよ〜ん|
+|あっぷでーと|だよ～ん| |アップデート|だよ~ん|
 |めっせんじゃ|では|ないから| |メッセンジャ|では|ないから|
 |じっこうした|ばあいは| |実行した|場合は|
 |ぶっくまーく|つーる|ばー| |ブックマーク|ツール|バー|
